docs/fuel-101/sequencer.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Sequencer
+category: Fuel 101
+---
+
+# What is a Sequencer?
+
+> A sequencer is a component utilized in Layer 2 scaling solutions, such as Optimistic Rollups and certain types of Zero-Knowledge Rollups to order transactions.
+
+Since rollups strategically leverage the consensus and data availability of another layer 1 chain, they depend on a sequencer for transaction ordering.
+
+By efficiently ordering transactions, sequencers allow Layer 2 solutions to significantly reduce transaction processing times and costs. This efficiency gain is essential for scaling blockchain networks and improving their usability for a wider range of applications, making the trade-off an integral part of optimizing blockchain performance.
```

Comment:
What's the tradeoff?

Reply:
I was implying layer 2s not having their own consensus layer and having to leverage Ethereum's. Will remove "making the trade-off an integral part of optimizing blockchain performance." since its obviously a bit confusing.